.github/workflows/codecov.yml

```diff
@@ -0,0 +1,28 @@
+name: Run CodeCov
+
+on: [push, pull_request]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.x']
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest pytest-cov
+        pip install -e .
+    - name: Generate Report
+      run: |
+        pytest --cov=canopen --cov-report=xml
+    - name: Upload Coverage to Codecov
+      uses: codecov/codecov-action@v3
```

Comment:
Could it perhaps be merged with the existing GitHub Python package action?

Reply:
Yes, if that's what you want, I can easily update the PR and integrate the upload action into the existing `pythonpackage.yml` instead of having a new one.

Reply:
Please integrate this, and add it into the existing pythonpackage.yml. This PR kinda slept?

Reply:
Done. But it does rely on the repo owner, e.g. @christiansandberg, to setup an codecov account and configure a token for the repo for it to work.